python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -306,18 +341,24 @@ export default function EditorContainer({
         {aiconfigState.prompts.map((prompt: ClientPrompt, i: number) => {
           return (
             <Stack key={prompt._ui.id}>
-              <PromptContainer
-                index={i}
-                prompt={prompt}
-                getModels={callbacks.getModels}
-                onChangePromptInput={onChangePromptInput}
-                onChangePromptName={onChangePromptName}
-                onRunPrompt={onRunPrompt}
-                onUpdateModel={onUpdatePromptModel}
-                onUpdateModelSettings={onUpdatePromptModelSettings}
-                onUpdateParameters={onUpdatePromptParameters}
-                defaultConfigModelName={aiconfigState.metadata.default_model}
-              />
+              <Flex mt="md">
+                <PromptMenuButton
+                  promptId={prompt._ui.id}
+                  onDeletePrompt={() => onDeletePrompt(prompt._ui.id)}
+                />
```

Comment:
Oh yea left comment (https://github.com/lastmile-ai/aiconfig/pull/665#issuecomment-1872171176) how I said this feels a bit weird it being on the left instead of right, but yea not a big deal

Reply:
But also why not put this within the prompt container?

Reply:
Just a design thing, it's not in the figma designs so just went with how it is in workbooks right now. We can update later